src/azul/queues.py

```diff
@@ -200,12 +200,12 @@ def wait_for_queue_level(self, empty: bool = True, num_bundles: int = None):
             queue_wait_time_elapsed = (time.time() - wait_start_time)
             queue_size_history.append(total_message_count)
             cumulative_queue_size = sum(queue_size_history)
-            if queue_wait_time_elapsed > timeout:
-                logger.error('The queue(s) are NOT at the desired level.')
-                return
-            elif (cumulative_queue_size == 0) == empty and len(queue_size_history) == deque_size:
+            if (cumulative_queue_size == 0) == empty and len(queue_size_history) == deque_size:
```

Comment:
Can you motivate this change a bit for me?

Reply:
Switching the order isn't strictly necessary, but if the queues get to zero, and this is logged, you would be surprised to see the log message that the queues are not at the desired level.

Reply:
Makes sense.